app/_src/guides/restrict-permissions-to-selected-namespaces-on-kubernetes.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Restrict permissions to selected namespaces on Kubernetes
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-demo = kuma | append: "-demo" %}
+{% assign kuma-another-demo = kuma | append: "-another-demo" %}
+
+By default, {{ Kuma }} deployed on Kubernetes has permission to observe and react to events from resources across the entire cluster. While this behavior simplifies initial setup and testing, it might be too permissive for production environments. Limiting {{ Kuma }}'s access to only necessary namespaces helps enhance security and prevents potential impact on unrelated applications.
```

Comment:
```suggestion
By default, {{ Kuma }} deployed on Kubernetes has permissions to observe and react to events from resources across the entire cluster. While this behavior simplifies initial setup and testing, it might be too permissive for production environments. Limiting {{ Kuma }}'s access to only necessary namespaces helps enhance security and prevents potential impact on unrelated applications.
```

Reply:
addressed